docs/error-api.asciidoc

```diff
@@ -122,7 +47,7 @@ Example error requests:
 
 [[payload-with-error]]
 [float]
-==== Payload with an Error
+===== Payload with an Error
```

Comment:
Same, those examples are still v1, we need to update them. Maybe you can just remove them for now.

Reply:
Removing until #1532 is ready.